tests/spec/s-xregexp-natives.js

```diff
@@ -508,13 +508,16 @@ describe('When overridden, String.prototype.replace()', function() {
 
                 it('should return the named backreference', function() {
                     expect('test'.replace(XRegExp('(?<test>t)', 'g'), ':${test}:')).toBe(':t:es:t:');
+                    expect('test'.replace(XRegExp('(?<test>t)', 'g'), ':$<test>:')).toBe(':t:es:t:');
 
                     // Backreference to a nonparticipating capturing group
                     expect('test'.replace(XRegExp('t|(?<test>t)', 'g'), ':${test}:')).toBe('::es::');
+                    expect('test'.replace(XRegExp('t|(?<test>t)', 'g'), ':$<test>:')).toBe('::es::');
                 });
 
```

Comment:
Can you add a test for mixing `${n}` and `$<n>` syntax in the same replacement string?